docs/FAQ.md

```diff
@@ -0,0 +1,40 @@
+---
+comments: true
+---
+# :kapitan-logo: FAQ
+## Why do I need Kapitan?
+
+With Kapitan, we worked to de-compose several problems that most of the other solutions are treating as one.
+
+1) ***Kubernetes manifests***: We like the jsonnet approach of using json as the working language. Jsonnet allows us to use inheritance and composition, and hide complexity at higher levels.
```

Comment:
Just jsonnet?

Reply:
Still lots of documentation is obsolete and needs updating